tools/PI/DevHome.PI/Controls/AddToolControl.xaml.cs

```diff
@@ -123,6 +142,10 @@ private void UpdateSampleCommandline()
         }
 
         var tool = GetCurrentToolDefinition();
+        if (tool is null)
+        {
+            return;
```

Comment:
I'm surprised this compiled.  Please add `return null` for consistency.

Reply:
Nope, it's a void function.